src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -990,73 +990,77 @@ namespace
         const float decay_start = light_object->GetDecayRadius(time);
         const int decay_exponent = light_object->GetDecayType();
 
-        // Create a color entity.
-        const std::string color_name =
-            insert_color(assembly, light_name + "_color", color);
-
-        // Get light from envmap.
-        INode* sun_node(nullptr);
-        BOOL sun_node_on(FALSE);
-        float sun_size_mult;
-        if (rend_params.envMap != nullptr &&
-            rend_params.envMap->IsSubClassOf(AppleseedEnvMap::get_class_id()))
+        // Skip exporting lights with zero intensity
+        if ((color[0] + color[1] + color[2]) > 0.0f && intensity > 0.0f)
```

Comment:
You can use `asf::is_zero()` on colors, spectra, vectors, etc.

Reply:
Done. Thanks for the tip.